ios/ElectronCash/electroncash_gui/ios_native/network_dialog.py

```diff
@@ -230,7 +230,7 @@ def refresh(self) -> None:
         self.lastPort = at(int(port))
         self.autoServerSW.on = bool(auto_connect)
 
-        host = network.interface.host if network.interface else _('None')
+        host = network.interface.host if network.interface else pgettext('Referencing server', 'None')
```

Comment:
`NameError` here -- can you update this PR to import `pgettext` from i18n up top? Looks like you forgot to. :)